lib/extensions/virtual_total_mixin.rb

```diff
@@ -108,3 +108,9 @@ def virtual_aggregate_arel(reflection, method_name, column)
     end
   end
 end
+
+module ActiveRecord
```

Comment:
I personally prefer `ActiveRecord::Base.send(:include, VirtualTotalMixin)` or something like https://github.com/ManageIQ/manageiq/blob/master/lib/extensions/descendant_loader.rb#L269 
When I read :point_up: it makes it feel like it's not okay to add anything else to `ActiveRecord::Base` here.